metering/metering-upgrading-metering.adoc

```diff
@@ -91,11 +92,12 @@ Many Pods rely on other components to function before they themselves can be con
 .Procedure (CLI)
 *  Check that all required Pods in the `openshift-metering` namespace are created:
 +
+[source,terminal]
 ----
 $ oc -n openshift-metering get pods
```

Comment:
This is completely out of scope for this PR, but I was looking at the rendered view on this, and we're also pretty inconsistent on how we specify a namespace for a particular `oc` command. In some instances, we use `oc --namespace ...`, others `oc -n ...` or `oc <command> <resource> -n ...` and it would be nice if we aimed towards some consistency for this.